docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -0,0 +1,431 @@
+service/teleport   LoadBalancer   10.100.37.171   a232d92df01f940339adea0e645d88bb-1576732600.us-east-1.elb.amazonaws.com   443:30821/TCP,3023:30801/TCP,3026:32612/TCP,3024:31253/TCP   4m21s
+
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/teleport   2/2     2            2           4m21s
+
+NAME                                  DESIRED   CURRENT   READY   AGE
+replicaset.apps/teleport-5cf46ddf5f   2         2         2       4m21s
+```
+
+### Set up DNS
+
+You'll need to set up two DNS `A` records: `teleport.example.com` for the web UI, and `*.teleport.example.com`
+for web apps using [application access](../../../application-access/introduction.mdx).
+
+Here's how to do this in a hosted zone with AWS Route 53:
+
+```bash
+# Change these parameters if you altered them above
+$ NAMESPACE=teleport
+$ RELEASE_NAME=teleport
+
+# Tip for finding AWS zone id by the domain name (replace your namespace)
+$ MYIP=$(kubectl --namespace ${NAMESPACE?} get service/${RELEASE_NAME?} -o jsonpath='{.status.loadBalancer.ingress[*].hostname}')
+$ MYZONE_DNS="example.com"
+$ MYZONE=$(aws route53 list-hosted-zones-by-name --dns-name=${MYZONE_DNS?} | jq -r '.HostedZones[0].Id' | sed s_/hostedzone/__)
+
+$ MYDNS="teleport.example.com"
+
+# Create a JSON file changeset for AWS.
+$ jq -n --arg ip ${MYIP?} --arg dns ${MYDNS?} '{"Comment": "Create records", "Changes": [
+  {"Action": "CREATE", "ResourceRecordSet": {"Name": $dns, "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}},
+  {"Action": "CREATE", "ResourceRecordSet": {"Name": ("*." + $dns), "Type": "A", "TTL": 300, "ResourceRecords": [{ "Value": $ip}]}}
+  ]}' > myrecords.json
+
+# Review records before applying.
+$ cat myrecords.json | jq
+# Apply the records and capture change id
+$ CHANGEID=$(aws route53 change-resource-record-sets --hosted-zone-id ${MYZONE?} --change-batch file://myrecords.json | jq -r '.ChangeInfo.Id')
+
+# Verify that change has been applied
+$ aws route53 get-change --id ${CHANGEID?} | jq '.ChangeInfo.Status'
+"INSYNC"
+```
+
+### Create a Teleport user
+
+At this point, you'll still need to create a user to be able to log into Teleport. This needs to be done on the Teleport auth server,
+so we can run the command using `kubectl`:
+
+```bash
+$ kubectl --namespace teleport exec deploy/teleport -- tctl users add test --roles=access,editor
+User "test" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport.example.com:443/web/invite/91cfbd08bc89122275006e48b516cc68
+
+NOTE: Make sure teleport.example.com:443 points at a Teleport proxy which users can access.
+```
+
+Load the user creation link to create a password and set up 2-factor authentication for the Teleport user via the web UI.
+
+You can follow our [Getting Started with Teleport guide](../../../getting-started.mdx#step-2-create-a-teleport-user-and-set-up-two-factor-authentication) to finish setting up your
```

Comment:
Let's move this to the bottom to the `Next steps`, so the article "flows". We dont' want to send the user elsewhere until they read the whole thing.